build/build-image/Dockerfile

```diff
@@ -14,7 +14,7 @@
 
 # This file creates a standard build environment for building Kubernetes
 
-FROM kube-build:cross
+FROM KUBE_BUILD_IMAGE_CROSS
```

Comment:
Oh, I see the `sed` below.  Is there a better way to do this?  At the very least, definitely add a comment because this pretty sneaky.